### ISSUE TITLE
confusion about the parameters of Normalize

### ISSUE THREAD

**Issue by zerens** [HaoMood/bilinear-cnn#12]
Regarding line 129-130 in bilinear_cnn_fc.py, I'm confused about the magic number in
```
Normalize(mean=(0.485, 0.456, 0.406),
                 std=(0.229, 0.224, 0.225))
```
where are these numbers from?

**Comment by HaoMood** [comment]
In PyTorch, all ImageNet pre-trained models expect input images normalized in the same way, i.e. mini-batches of 3-channel RGB images of shape (3 x H x W), where H and W are expected to be at least 224. The images have to be loaded in to a range of [0, 1] and then normalized using mean = [0.485, 0.456, 0.406] and std = [0.229, 0.224, 0.225]. 

https://pytorch.org/docs/stable/torchvision/models.html
